### PR TITLE
Make anchor links to each video in the docs

### PR DESCRIPTION
**Pull request by ankur-gos**
Pull request for issue #4069
Added anchor links to each rst file containing a video.

### CODE REVIEW

File: docs/source/user/commands.rst [hunk 1]
```diff
@@ -13,6 +13,8 @@ JupyterLab commands:
    :align: center
    :class: jp-screenshot
 
+.. _access_pallette:
```
Comment (ian-r-rose) [inline docs/source/user/commands.rst]: Typo here: "pallette" ==> "palette"

**Comment by jasongrout** [comment]
Thanks!

Just curious: does this insert the little anchor marker in the html, so it's easy to get a URL for the anchor?

**Comment by ankur-gos** [comment]
@jasongrout I'm not completely sure, but on inspecting the output, looks like you can get the label by looking at the <p> tag above the video and using the id on that <p> field. So if the id = create-notebook, the url would be something like mybaseurl.com/specificpage#create-notebook.

**Comment by ankur-gos** [comment]
@ian-r-rose Fixed that typo, thanks!

**Comment by ian-r-rose** [comment]
Sorry @ankur-gos, one more thing: it looks like Sphinx normalizes underscores in anchors to dashes (`-`), so all of the multi-word anchors here get converted. For instance `#create_console` becomes `#create-console`. I think this will be kind of confusing when referring to these anchors. Can we change them to dashes?

**Comment by jasongrout** [comment]
> I'm not completely sure, but on inspecting the output, looks like you can get the label by looking at the <p> tag above the video and using the id on that <p> field. So if the id = create-notebook, the url would be something like mybaseurl.com/specificpage#create-notebook.

Would it be too distracting to have a really subtle header then, something you could click on to get the URL? That may break up the flow of the docs too much, so I don't know...

**Comment by ian-r-rose** [comment]
@jasongrout I was thinking about that as well. I'm not sure that we want to restructure the docs to make linking slightly easier. If one is linking from *within* the JupyterLab documentation, it is fairly easy to inspect the anchor labels, since you already have them open in a text editor. If you are linking from outside, it's a bit more work...

**Comment by jasongrout** [comment]
> I'm not sure that we want to restructure the docs to make linking slightly easier.

*If* it's unobtrusive, it would be great to have. It it interrupts the flow too much, never mind, I can do the difficult way...

**Comment by ankur-gos** [comment]
@ian-r-rose Changing to dashes is no problem. I can also change existing anchor links to dashes if you're ok with that?

**Comment by ian-r-rose** [comment]
That would be great!

**Comment by ian-r-rose** [comment]
Hm, that's odd, I'm not sure. You can try kicking the CI by closing and then reopening the PR.

**Comment by ian-r-rose** [comment]
It looks like the travis build is broken for unrelated reasons. Forging ahead, since this is clearly not the problem. Thanks @ankur-gos !

**Pull request merged**